squall/squall_vm.hpp

```diff
@@ -38,20 +38,26 @@ class VM {
         HSQUIRRELVM vm = handle();
         sq_setforeignptr(vm, &klass_table_);
 
-        // root table取得
         sq_pushroottable(vm);
         sq_getstackobj(vm, -1, &root_);
         sq_pop(vm, -1);
 
         root_table_.reset(new TableBase(vm, root_));
     }
-    ~VM() { sq_setforeignptr(handle(), 0); }
+    ~VM() { sq_setforeignptr(handle(), 0); } 
 
-    template <class R, class... T>
-
-    R call(const string& name, T... args) {
+#ifdef _MSC_VER
+    //Avoiding Visual C ++ bugs
+    template <class R, typename... Args>
+    R call(const string& name, Args... args) {
+        return root_table_.get()->call<R>(name, args...);
```

Comment:
Latest visual c++ has bugs in variadic templates.
This is hack for avoiding it.

Reference: https://developercommunity.visualstudio.com/content/problem/169107/syntax-error-in-c-variadic-template-after-latest-u.html